packages/api/src/router/board.ts

```diff
@@ -48,29 +49,45 @@ const filterUpdatedItems = <TInput extends { id: string }>(
 
 export const boardRouter = createTRPCRouter({
   getAll: publicProcedure.query(async ({ ctx }) => {
-    return await ctx.db.query.boards.findMany({
+    const myPermissions = await ctx.db.query.boardPermissions.findMany({
```

Comment:
``my`` is non descriptive because the user context is unknown. Use a better name